### PR TITLE
dev-qt/qtsql: Add IUSE=mariadb

### PR DESCRIPTION
**Pull request by a17r**

### CODE REVIEW

File: dev-qt/qtsql/qtsql-5.15.2-r1.ebuild [hunk 1]
```diff
@@ -41,17 +43,28 @@ QT5_GENTOO_PRIVATE_CONFIG=(
 
 PATCHES=( "${FILESDIR}/${P}-mysql-use-pkgconfig.patch" )
 
+src_prepare() {
+	qt5-build_src_prepare
+	if use mariadb; then
+		sed -e "/type.*pkgConfig.*mysqlclient/s/mysqlclient/libmariadb/" \
+		-i src/plugins/sqldrivers/configure.json || die
```
Comment (Pesa) [inline dev-qt/qtsql/qtsql-5.15.2-r1.ebuild:50]: can you indent this line since it's a continuation?

**Review by Pesa — COMMENTED** [review]
the revbump should be in the second commit, no?

**Comment by a17r** [comment]
> the revbump should be in the second commit, no?

Yes, would have gone to gentoo.git instead of course, but I hope with a recent fix to akonadi we can avoid pt.2 of this PR altogether. Supporting mariadb-connector-c here is slightly messy as there are several revdeps depending on qtsql[mysql] that would need change/figuring out.

Do you think it is worth keeping the pkgconfig patch? It is certainly much cleaner than running mysql_config.

**Comment by Pesa** [comment]
> Do you think it is worth keeping the pkgconfig patch? It is certainly much cleaner than running mysql_config.

It would definitely be cleaner. OTOH if it's not broken, why bother? Up to you, I have no preference.

**Pull request closed without merging**